core/domain/takeout_service.py

```diff
@@ -46,6 +47,32 @@ def get_models_which_should_be_exported():
             base_models.EXPORT_POLICY.CONTAINS_USER_DATA]
 
 
+def export_all_profiles_data_by_parent_user(parent_user_id):
+    """Exports selected models according to model defined export_data functions
+        for all profile users linked to the parent_user of the given id.
```

Comment:
deindent by 4

Reply:
Ditto. Removed the functions now as per the other comment.